live.ts

```diff
@@ -112,6 +120,10 @@ export const withLive = (
     // TODO add custom middleware optional
 
     // Prepare ctx.state for page handlers
+    ctx.state.loadFlags = async () => {
+      ctx.state.flags = await loadFlags(req, ctx) as Flag[];
```

Comment:
Doesn't `loadFlags` already change the `ctx.state.flags`?

https://github.com/deco-cx/live/pull/47/files#diff-6a92ff4deddc2565bf7e9c8a01cb25b2083b9a782c6129d23b5faf13cd1d14a1R77

Reply:
😴